### PR TITLE
Add Mergin CLI

### LINKED ISSUE #4: Add mergin commandline tool
see https://twitter.com/lutraconsulting/status/1403375915795726344

### PR DESCRIPTION
**Pull request by ivanoats**
Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ A collection of geospatial programs with commandline interface. The starting poi
 - [geodiff](https://github.com/lutraconsulting/geodiff) - Handle differences in geospatial data.
 - [GeoGig](http://geogig.org/) - Geospatial version control.
 - [Kart](https://kartproject.org) - Geospatial version control.
+- [mergein](https://github.com/lutraconsulting/mergin-py-client) - Python client module for access to [Mergin](https://public.cloudmergin.com) and a command-line tool for access to data stored in Mergin.
```
Comment (JakobMiksch) [inline README.md:86]:
```suggestion
- [mergin](https://github.com/lutraconsulting/mergin-py-client) - Python client module for access to [Mergin](https://public.cloudmergin.com) and a command-line tool for access to data stored in Mergin.
```

Comment (JakobMiksch) [reply README.md:86]: Looks good. One typo.

**Review by JakobMiksch — CHANGES_REQUESTED** [review]

**Review by JakobMiksch — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ivanoats) [reply README.md]: thanks for catching that and merging!

**Comment by JakobMiksch** [comment]
@ivanoats thanks for the contribution

**Pull request merged**